coreos-installer

```diff
@@ -591,6 +591,18 @@ write_image_to_disk() {
     udevadm settle
 }
 
+############################################################
+#Make call home HTTP(s) call as the final step
+############################################################
+call_home() {
+    CALLHOME_URL=$(cat /tmp/callhome_url)
+    if [ ! -z "$CALLHOME_URL" ]
+      echo "Calling home: $CALLHOME_URL" >> /tmp/debug
+      curl -skf -o /dev/null --data-binary @/tmp/debug "$CALLHOME_URL" >/tmp/debug 2>&1
```

Comment:
Is there a particular reason for this to be an insecure request?

As well, why the silent failure? If a user has specified a call home URL it seems like not being able to dial home is a potentially critical failure.

Reply:
> Is there a particular reason for this to be an insecure request?

Yes, there is. Where would I get a server certificate from? Or is there a mechanism to inject/embed certificates in the initramdisk? Then I will happily remove that, however this will force users to rebuild initramdisk everytime. Pulling it from network does not make any sense to me.

> As well, why the silent failure?

This was a copy-paste error. No particular reason.

Reply:
Not easily.  However, on a related topic for OpenShift 4 the installer sets things up so that the Ignition served to the device is just a (URL, cert) pair.  See e.g. https://github.com/openshift/machine-config-operator/pull/759
However, we can't quite fit a cert into a kernel command line. 

Maybe what we should do is have e.g. `coreos.inst.callhome_cert_fingerprint` and trust the cert matching that fingerprint.  But eh, don't need to do it in this patch.

Reply:
I'll do what you say, however I also lean towards just waiving this.

Reply:
I was more thinking along the lines of forcing the user to specify HTTP endpoints to convey that the call is insecure rather than accepting HTTPS endpoints and ignoring cert failures.